common/FileUtils.java

```diff
@@ -238,19 +241,36 @@ public Message readProtoBinaryUnchecked(String path, Message.Builder builder) {
    * Copies a directory to another directory. Creates target directory if needed and doesn't copy
    * symlinks.
    */
-  public void copyDirectoryToDirectory(String source, String destination, String ignored)
+  public void copyDirectoryToDirectory(String source, String destination, String... ignored)
       throws IOException {
     final Path sourcePath = fileSystem.getPath(source);
     final Path targetPath = fileSystem.getPath(destination);
+
+    List<String> allFilesAndFoldersForIgnore = new ArrayList<>();
```

Comment:
s/allFilesAndFoldersForIgnore/filesAndFoldersToIgnore

Reply:
Deleted it.